### ISSUE TITLE
accessibility: when exporting a note to PDF, tags are not transferred

### ISSUE THREAD

**Issue by ericdq** [laurent22/joplin#14086]
### Operating system

Windows

### Joplin version

3.5.11

### Desktop version info

Joplin 3.5.11 (prod, win32)

Dispositivo: win32, 11th Gen Intel(R) Core(TM) i5-11400 @ 2.60GHz
ID del Cliente: dfd34631c9ab4a0c9f5aeaa86dcb0faf
Versión de la Sincronización: 3
Versión del Perfil: 49
Llavero Soportado: Sí
ID de instancia alternativa: -

Revisión: e75875c

Backup: 1.5.1
joplin2jira: 1.0.0

### Current behaviour

When I export a Markdown note to PDF, tags such as headings, lists, links, language, etc. are not preserved. However, if I export to HTML, it works correctly. I do not know how to fix this. In some OpenOffice documents, if I format them there, I can export them to PDF correctly.

Although it has no bearing on the issue, I am copying the details of my version of Joplin. This error occurs in both stable and beta versions.

**Comment by personalizedrefrigerator** [comment]
Thank you for the issue report!

For reference, Joplin is using the Electron `webContents.printToPdf` API to create a PDF:

https://github.com/laurent22/joplin/blob/a3bf0cfdebf1dd97d8d520571d4a3cbe6fd9acc6/packages/app-desktop/InteropServiceHelper.ts#L91

It looks like Electron provides an (experimental) [`generateTaggedPDF`](https://www.electronjs.org/docs/latest/api/web-contents#contentsprinttopdfoptions) setting that might help here. The property is described with:
> As this property is experimental, the generated PDF may not adhere fully to PDF/UA and WCAG standards.

If this is the only reason that `generateTaggedPDF` is experimental, it may make sense for Joplin to enable this by default (without a setting to disable it).

**Comment by ericdq** [comment]
Can I activate it in some way in my installation, or will it be possible to try it when a new version comes out?

**Comment by personalizedrefrigerator** [comment]
> Can I activate it in some way in my installation, or will it be possible to try it when a new version comes out?

It should be possible to try this when https://github.com/laurent22/joplin/pull/14111 (or a similar pull request) is merged and released.
